experimental/traffic-portal/src/app/shared/obscured-text-input/obscured-text-input.component.html

```diff
@@ -0,0 +1,31 @@
+<!--
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<div tabindex="0" class="mat-select-trigger" (focusin)="focus()" (focusout)="blur($event)">
```

Comment:
Not 100% that this line is the issue but when tabbing to the obscured field, a non-functional box gets focus first (doesn't appear to toggle the visibility either) requiring an extra tab.

Reply:
It could be that. When I leave off the `tabindex` attribute in this particular component, the `focusin` and `focusout` events never fire. However, when trying to replicate the problem with a CMVE I couldn't and didn't need `tabindex` to get those events working.

Reply:
nvm, I can't reproduce it now, either. It must've been something about the form control being partially unfinished at the time or something.